.github/workflows/validate-chart.yaml

```diff
@@ -36,7 +36,8 @@ jobs:
           chmod 755 kube-score
 
       - name: Kube-score generated manifests
-        run: helm template  --values charts/.ci/values-kube-score.yaml charts/* | ./kube-score score -
+        run: |
+          helm template  --values charts/.ci/values-kube-score.yaml charts/* | ./kube-score score -
```

Comment:
This step was failing with:

```
Run helm template  --values charts/.ci/values-kube-score.yaml charts/* | ./kube-score score -
Error: open charts/.ci/values-kube-score.yaml: no such file or directory
/home/runner/work/_temp/fa25a86f-4d07-4fb2-a01a-4260a335971a.sh: line 2: --ignore-test: command not found
```

Reply:
You'd need to either (1) add `\` at the end of each line in the command or (2) revert it to use `run:` instead of `run: |`. I personally prefer (2) as we might have been using `run: ` to avoid (1) in the first place 😃